### PR TITLE
toaster_configuration: Add MGLS_LICENSE_FILE from to config

### PR DESCRIPTION
**Pull request by sharidas**
Add MGLS_LICENSE_FILE to toaster configuration

Signed-off-by: Sujith Haridasan <Sujith_Haridasan@mentor.com>

### CODE REVIEW

**Comment by sharidas** [comment]
@Noor-Ahsan @skrishnakar @kergoth Kindly review my change.

File: scripts/toaster_configuration [hunk 1]
```diff
@@ -24,6 +24,7 @@ create_toaster_configuration() {
 DISTRO=$(grep "DISTRO =" "$BUILDDIR/conf/local.conf"  | awk {'print $3'} | sed -e s:\'::g | head -n 1)
 MACHINE=$(grep -i "MACHINE ??=" "$BUILDDIR/conf/local.conf" | awk {'print $3'} | sed -e s:\"::g)
 EXTERNAL_TOOLCHAIN=$(grep -i "^EXTERNAL_TOOLCHAIN" "$BUILDDIR/conf/local.conf" | awk {'print $3'})
+MGLS_LICENSE_FILE=$(env | grep "MGLS_LICENSE_FILE=" | sed -e 's:^.*=::' | sed -e 's:\:$::')
```
Comment (kergoth) [inline scripts/toaster_configuration]: This is completely pointless complexity. Is there a reason you aren't just using $MGLS_LICENSE_FILE? 'env' has access to the same env you already have, so you're spawning a child and extracting a variable you already have in your own environment.

Comment (kergoth) [inline scripts/toaster_configuration]: Just remove this line entirely.

Comment (sharidas) [inline scripts/toaster_configuration]: Sure. I will remove that line. Thanks for pointing out.

**Pull request merged**